src/api/variants/collection.js

```diff
@@ -147,6 +148,7 @@ module.exports = class VariantCollection extends EntityCollection {
                 order: viewFile.order,
                 dir: opts.dir,
                 isHidden: viewFile.isHidden,
+                notes: notes && notes.isFile ? notes.getContentSync() : undefined,
```

Comment:
- pass the readme file as `readme` key, since at this stage Fractal considers notes as coming from config file & readme as separate file
- the isFile/getContentSync would be better inside Variant class `get notes()` getter, so the implementation would be more similar to Component class